sandpack-client/src/types.ts

```diff
@@ -59,6 +59,14 @@ export interface ClientOptions {
    * to retrieve npm packages from your own npm registry.
    */
   customNpmRegistries?: NpmRegistry[];
+
+  /**
+   * Configuration for hidden `link` and `script` tags.
+   */
+  hiddenHeadTags?: {
```

Comment:
I'm wondering if we can use `externalResources` (which is for the same purpose but works differently on the runtime client), instead of creating a new one that kind of does the same thing. 

Plus, the export button could carry this value (`externalResources`) to the sandbox on CodeSandbox.io, which would make the experience smoother.

Reply:
You can see how csb.io implements `externalResources` [here](https://github.com/codesandbox/codesandbox-client/blob/8051314f7dfb71ac23a9a5dfc9ccf39f79508728/packages/app/src/sandbox/external-resources.ts#L85) and get some inspiration

Reply:
Great! 

> I think the idea of adding the <!DOCTYPE html> tag if it doesn't exist should be easy enough. 

If I understood correctly if a document misses the document tag it might affect the CSS behavior, which is not necessarily true for `html`, `head`, and `body` tags. So, I'd consider adding only what's needed to ensure a standard experience for everyone, meaning adding the doctype html.

> On the note about carrying over the externalResources value to CodeSandbox.io, it looks like that value might not be used when you click the Open Sandbox button. I tried it with the existing External Resources story which uses the runtime client and the tailwind script doesn't seem to carry over when I click the button.

This is correct. It's not properly working now, due to some backend issues, but let's consider that in the future we will use it.

Reply:
@danilowoz I just pushed an updated version with the following changes:
1. Using the `externalResources` prop
2. Injecting the `<!DOCTYPE html>` tag if it doesn't exist

Here's an example with the setup: https://codesandbox.io/s/sandpack-hidden-head-tags-external-resources-example-iwcny8?file=/src/App.js

**Note:** That example also pulls in an external Tailwind CSS file. That should keep things in-line with how `externalResources` works for the Runtime Client.

Let me know what you think.